### PR TITLE
Docs - ainft, event

### PR DESCRIPTION
**Pull request by woomurf**
- Add ainft and event module simple documentation.

### CODE REVIEW

File: src/event.ts [hunk 1]
```diff
@@ -129,24 +159,49 @@ export default class Event extends AinftBase {
     return this.sendRequest(HttpMethod.PUT, trailingUrl, body);
   }
 
+  /**
+   * Returns a list of TaskTypes to use for events.
```
Comment (platfowner) [inline src/event.ts:163]: Return / Returns, Provide / Provides ... (3인칭 단수 / 복수) 통일되면 좋을 것 같습니다~

**Review by platfowner — APPROVED** [review]
LGTM with some comments.
Thanks!

File: src/event.ts (hunk 1 above)
Comment (woomurf) [reply src/event.ts:163]: Done!

**Comment by woomurf** [comment]
Thanks for the review!

**Pull request merged**